tests/unit/core/statistics/datatypes/test_floattracker.py

```diff
@@ -13,6 +15,28 @@ def test_values_are_min_max():
     assert first.sum == sum(vals1)
 
 
+def test_np_float():
+    first = FloatTracker()
+    float32_array = np.array([1.0, 2.0, 3.0], dtype=np.float32)
+    for val in float32_array:
+        first.update(val)
+
+    assert first.count == len(float32_array)
+    assert first.max == max(float32_array)
+    assert first.min == min(float32_array)
+    assert first.sum == sum(float32_array)
+
+    second = FloatTracker()
```

Comment:
suggest naming something like 'tracker_2' to avoid overloaded term with unit of time 'second'

consider using parameterized test inputs e.g.
https://github.com/whylabs/whylogs/blob/mainline/tests/unit/core/types/test_typeddataconverter.py#L38-L39

Reply:
How would you use parametrized tests in this case? Would it be to replace the setup like `    float32_array = np.array([1.0, 2.0, 3.0], dtype=np.float32)`

Reply:
You could do that and parameterize the data, or you could parameterize only the dtype:
`_TEST_NP_FLOAT_DTYPES = [np.float16, np.float32, np.float64, ]`

But what you have looks good and fixes a bug, let's merge.